### ISSUE TITLE
Not A Feature Request

### ISSUE THREAD

**Issue by spacehampsters** [Mgamerz/ME3Explorer#310]
Hello! I wasn't sure whether to make this a feature request or bug seeing as it's neither. I'm fairly new to modding and made my own headmesh for my Shepard (following the tutorial for armor editing in blender). I'm working in ME3. It went really well until I imported the mesh into Meshplorer (replacing the defualt femshep mesh). The eyelash texture took up the entirety of her face. When I deleted it (the eyelash texture) the area was just replaced with blue and white checkers. I couldn't find any tutorials here or online regarding the issue and I'm fairly confused about textures in Legendary Explorer in general. Hopefully you could give me some advice, but if not that's okay!

**Comment by Padme4000** [comment]
Hi its because the material order changes in blender so you can either change the material order in blender to match the original order you see in meshplorer by clicking on a material in blender and using the up and down arrows.

Or just swap the material numbers around in meshplorer to match the new material order of your mesh in blender.

This mini tutorial [here](https://github.com/ME3Tweaks/LegendaryExplorer/wiki/A-guide-for-Multiple-Material-Meshes) should help

**Comment by spacehampsters** [comment]
Thank you!

**Comment by spacehampsters** [comment]
I've been attempting to replace the defualt femshep hair. I managed to replace the mesh but in game parts of the hair are missing. I have a couple theories as to why this might be. I haven't been able to replace HMF_HIR_SHEPARD_MDL_Remaster because I'm unsure of how to reduce the bone number to 1 in order to import it. My other idea is that it's because I deleted many bones in order to import it. Lastly I think it might be a texture thing, but I'm unsure how to fix it if that's the case as the defualt femshep hair doesn't have any textures to replace. Hoping someone knows the answer, and I'm sorry for asking so many questions!
